src/main/java/frc/robot/commands/LocateTarget.java

```diff
@@ -9,59 +9,77 @@
 // it from being updated in the future.
 
 package frc.robot.commands;
-import edu.wpi.first.wpilibj.command.Command;
-import frc.robot.Robot;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.subsystems.Targeting;
+// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
 
 /**
  *
  */
-public class LocateTarget extends Command {
+public class LocateTarget extends CommandBase 
+{
     // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=VARIABLE_DECLARATIONS
+        private final Drivetrain m_drivetrain;
+ 		private final Targeting m_targeting;
     // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=VARIABLE_DECLARATIONS
-
-    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTOR
-    public LocateTarget() {
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTOR
+    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
+    
+    public LocateTarget(Drivetrain drivetrain, Targeting targeting) 
+    {
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
         // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=VARIABLE_SETTING
 
         // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=VARIABLE_SETTING
         // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
-        requires(Robot.drivetrain);
+
+        m_drivetrain = drivetrain;
+        addRequirements(m_drivetrain);
+        m_targeting = targeting;
+        addRequirements(m_targeting); // Is this needed?
```

Comment:
@BobTheLegend Do you know if LocatTarget requires the targeting subsystem? This might cause other things not to work